### ISSUE TITLE
Feature suggestion

### ISSUE THREAD

**Issue by yejinson** [th3-z/fb2k-mini-player#4]
1、Dragging moves the player without holding middle mouse.
2、Hide to the edge of the screen and automatically show when the mouse is moved over it.
3、Automatically display a larger image when the mouse is over an album image, just like Deskband Controls (a foobar component).
4、Add button to close foobar

**Comment by z3ath** [comment]
I think middle mouse to move was used to avoid accidental movement, there may have been a technical reason too but I can't remember. I agree it's not a user friendly choice, I'll investigate this point.

On point two and three, I'm very hesitant to add these features. The app is designed to be simple and unobtrusive, I don't want it jumping around every time I move the mouse near it.

If we want a close button, I think the anticipated behaviour would be closing the miniplayer and leaving foobar open. You can click the album artwork to focus foobar and close it normally, I find this to be a satisfactory method.

**Comment by z3ath** [comment]
On point two, I think having the opacity reduced until mouse over might be an idea

**Comment by yejinson** [comment]
> I think middle mouse to move was used to avoid accidental movement, there may have been a technical reason too but I can't remember. I agree it's not a user friendly choice, I'll investigate this point.
> 
> On point two and three, I'm very hesitant to add these features. The app is designed to be simple and unobtrusive, I don't want it jumping around every time I move the mouse near it.
> 
> If we want a close button, I think the anticipated behaviour would be closing the miniplayer and leaving foobar open. You can click the album artwork to focus foobar and close it normally, I find this to be a satisfactory method.

Thanks for your reply, maybe I didn't make it clear, what I meant was to use the left mouse button instead of the middle mouse button. Points 2 and 3 are no problem, because other players are like this, adding the close button is mainly for convenience, otherwise when you exit the player, you need to close it twice.

**Comment by yejinson** [comment]
5、Need more detailed color settings, because when I try to change the background color, it doesn't work at all, I need to change the black part to blue and the text part to black.<img src='https://s2.loli.net/2022/04/25/OGD3IaTC1qQxV8Z.png' title='quicker_a32b844a-11b4-4355-8def-4a288f8c4f32.png' />

**Comment by z3ath** [comment]
Yeah, I understand the request. I made this choice a long time ago and I'm pretty sure that was the behaviour I originally wanted but there was some good reason that I didn't use left mouse.

I'll investigate and get back to you with either the change or detail exactly why it's like that.

For point five, please send a screenshot with settings `fg_col=#000000`, `alpha=1.0` and `bg_col=#0000ff`.

**Comment by yejinson** [comment]
> Yeah, I understand the request. I made this choice a long time ago and I'm pretty sure that was the behaviour I originally wanted but there was some good reason that I didn't use left mouse.
> 
> I'll investigate and get back to you with either the change or detail exactly why it's like that.
> 
> For point five, please send a screenshot with settings `fg_col=#000000`, `alpha=1.0` and `bg_col=#0000ff`.
Well, too few settings options. <img src='https://s2.loli.net/2022/04/25/cot5HOy2CbkXB8P.png' title='quicker_c35fd2a6-a7d5-414e-9f9d-dfb4d79b24e9.png' />

**Comment by z3ath** [comment]
I'd like to stick to only foreground and background colour for the sake of simplicity but I can certainly make the button background colours adjust to the `bg_col`setting if they aren't doing that already.
